hieradata/role/jenkins-slave.yaml

```diff
@@ -0,0 +1,154 @@
+---
+psick::pre::windows_classes:
+  chocolatey: chocolatey
+psick::base::windows_classes:
+  packages: psick::packages
+  openssh: psick::openssh
+psick::profiles::windows_classes:
+  features: psick::windows::features
+  registry: psick::windows::registry
+  services: psick::windows::services
+  puppet: psick::puppet::pe_agent
+  reboot: psick::reboot
+# FIXME: Why commented? Do we need them?
+#  firewall: windows_firewall
+#  users: psick::users
+
+# Do not try to install openssh on Windows via tp
+psick::openssh::use_tp: false
+
+psick::packages::resource_default_arguments:
+  provider: chocolatey
+psick::packages::packages_list:
+  - "chocolatey"
+  - "tortoisesvn"
+  - "putty.install"
+  - "git"
+  - "visualstudio2017enterprise"
```

Comment:
Please remove `visualstudio2017-workload-netweb` and `visualstudio2017enterprise`, and install `webdeploy`.